storage/storage-proto/src/tests.rs

```diff
@@ -41,9 +43,6 @@ test_conversion!(
 test_conversion!(test_save_transactions_request, SaveTransactionsRequest);
 test_conversion!(test_get_transactions_request, GetTransactionsRequest);
 test_conversion!(test_get_transactions_response, GetTransactionsResponse);
-test_conversion!(test_tree_state, TreeState);
-test_conversion!(test_startup_info, StartupInfo);
-test_conversion!(test_get_startup_info_response, GetStartupInfoResponse);
```

Comment:
no way to keep?

Reply:
OIC we are deprecating GRPC.

@bors-libra delegate+

Reply:
too much work. Not worth it.